recipes/vmtk/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "vmtk" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vmtk/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ec52357bf6dfd975eaae592b6b70db67c7767a01278386373588a29cf52997bb
+
+build:
+  number: 0
+  features:
+    - vc14  # [win and (py35 or py36)]
+  skip:
+    - True  # [win and py27]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkgconfig              # [unix]
+    - cmake
+    - git
+    - ninja
+    - setuptools             # [win]
+
+  host:
+    - python {{ python }}
+    - future
+    - vtk {{ vtk }}
+    - itk {{ itk }}
+    - numpy <=1.13
+    - h5py
+    - setuptools              # [win]
+    - mesa                    # [linux]
+    - joblib
+
+  run:
+    - python {{ python }}
+    - future
+    - vtk {{ vtk }}
+    - itk {{ itk }}
+    - numpy <=1.13
+    - h5py
+    - mesa                    # [linux]
+    - joblib
+
+test:
+  source_files:
+    - build/tests/*
+  requires:
+    - pytest
+  commands:
+    - pytest ./build/tests/
+
+about:
+  description: |
+    The Vascular Modeling Toolkit is a collection of libraries and tools for 3D reconstruction, geometric analysis, mesh generation and surface data analysis for image-based modeling of blood vessels. VMTK can be used via its standalone interface, included as a Python or C++ library, or as an extension to the medical image processing platform 3D Slicer. By providing multiple user interfaces with various requirments of technical ability, VMTK aims to be usable by anyone with an interest in medical image processing; be they clinicians, researchers, industries, or educational institutions.
+  home: www.vmtk.org
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: "vmtk - the Vascular Modeling Toolkit. See www.vmtk.org/ for details."
+  doc_url: http://www.vmtk.org/documentation/
+  dev_url: https://github.com/vmtk/vmtk
+
+extra:
+  recipe-maintainers:
+    - rlizzo
+    - lantiga
+    - aslakbergersen
+    - hkjeldsberg
```

Comment:
The other potential maintainers need to confirm they are ok. They should leave a note here.

@rlizzo @lantiga @hkjeldsberg  for viz

Reply:
@hkjeldsberg @beckermr Looks great to me, thanks for this! I'm ok with being listed as a maintainer of course.

Reply:
This looks good to me as well! I'm good with being listed as a maintner in addition.

Reply:
@hkjeldsberg Are you OK here?

Reply:
We are still waiting to hear from @hkjeldsberg

Reply:
or you can remove @hkjeldsberg from the list

Reply:
I can confirm I'm ok with being listed as a maintainer!

Reply:
huzzah!